<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_tab</class>
 <widget class="QWidget" name="settings_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#top_widget {
background-color: rgb(54, 71, 79);
}
#header {
background-color: rgb(200, 200, 200);
}

.QRadioButton, .QPushButton {
	color: rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;
	border: null;
	padding: 10%;
}

#table {
background-color: rgb(241, 244, 249);
}

.QRadioButton:hover, .QPushButton:hover{
background-color: rgb(69, 75, 84);
}

#rigth_widget {
	background-color: rgb(39, 50, 56);
}

.QLabel {
	color: rgb(136, 139, 145);
	font: 700 9pt &quot;Segoe UI&quot;;
	padding: 10%;
}

#scrollAreaWidgetContents {
background-color: rgb(241, 244, 249);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="top_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/disk.svg</normaloff>icons/disk.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>19</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="rigth_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>СМЕНИТЬ ПАРОЛЬ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_manager">
        <property name="text">
         <string>Менеджер App</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_postgres_manager">
        <property name="text">
         <string>Менеджер DB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_admin">
        <property name="text">
         <string>Админ App</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_postgres_admin">
        <property name="text">
         <string>Админ DB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>БАЗА ДАННЫХ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_archive">
        <property name="text">
         <string>Архивировать</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="settings">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>635</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
